tests/trainer/optimization/test_optimizers.py

```diff
@@ -170,57 +214,85 @@ def test_optimizer_return_options():
     assert optim == [opt_a, opt_b]
     assert len(lr_sched) == len(freq) == 0
 
-    ref_lr_sched = dict(
+    # opt tuple of 2 lists
+    model.configure_optimizers = lambda: ([opt_a], [scheduler_a])
+    optim, lr_sched, freq = trainer.init_optimizers(model)
+    assert len(optim) == len(lr_sched) == 1
+    assert len(freq) == 0
+    assert optim[0] == opt_a
+    assert lr_sched[0] == dict(
         scheduler=scheduler_a,
         interval='epoch',
         frequency=1,
         reduce_on_plateau=False,
         monitor=None,
         strict=True,
+        opt_idx=None,
         name=None,
     )
 
-    # opt tuple of 2 lists
-    model.configure_optimizers = lambda: ([opt_a], [scheduler_a])
-    optim, lr_sched, freq = trainer.init_optimizers(model)
-    assert len(optim) == len(lr_sched) == 1
-    assert len(freq) == 0
-    assert optim[0] == opt_a
-    assert lr_sched[0] == ref_lr_sched
-
     # opt tuple of 1 list
     model.configure_optimizers = lambda: ([opt_a], scheduler_a)
     optim, lr_sched, freq = trainer.init_optimizers(model)
     assert len(optim) == len(lr_sched) == 1
     assert len(freq) == 0
     assert optim[0] == opt_a
-    assert lr_sched[0] == ref_lr_sched
+    assert lr_sched[0] == dict(
```

Comment:
This is undoing the usage of `ref_lr_sched` (see diff)

Probably comes from a messed merge

Reply:
@hemildesai to me looks like this was already there before I rebased. or did I overwrite something here? 
not sure what's the right thing to do here

Reply:
Yeah I think this was there before the merge. The `ref_lr_sched` is replaced with a custom dict for the schedulers because the dict for each lr scheduler in the test differs in one or two places.